src/api/index.ts

```diff
@@ -122,15 +129,22 @@ export class GithubDeploymentsApiClient implements GithubDeploymentsApi {
       deployments: GraphQlDeployment[];
     };
 
-    const statuses: statusesResponse = await octokit<statusesResponse>(statusesQuery, {
-      deploymentNodeIds: params.deploymentNodeIds
-    });
+    const statuses: statusesResponse = await octokit<statusesResponse>(
+      statusesQuery,
+      {
+        deploymentNodeIds: params.deploymentNodeIds,
+      },
+    );
 
     const formatStatuses = (d: statusesResponse) => {
       const formatted: RestDeploymentStatus[] = [];
       for (let i = 0; i < d.deployments.length; i++) {
         let deployment = d.deployments[i];
-        let status = deployment.statuses.nodes[0];
+
+        let status = deployment.statuses.nodes.filter(
+          s => s.state.toUpperCase() != 'INACTIVE',
+        )[0];
```

Comment:
This is probably o.k., but there's a hidden assumption that there will always be at least one status whose state isn't `'INACTIVE'`. If for some weird reason there's only one status and it's inactive, (or if the statuses list is initially empty), this will have an exception on the `[0]` indexing of an empty array.